client/admin.html

```diff
@@ -150,15 +154,24 @@ <h1 class="center">Admin Panel</h1>
 						<th>
 							<select id="branch-filter">
 								<option value="*">All types</option>
-								{{#each branchNames}}
-									<option value="{{this}}">{{this}}</span>
-								{{/each}}
+								<optgroup label="Application">
+									{{#each settings.branches.application}}
+										<option value="application-{{this.name}}">{{this.name}}</option>
+									{{/each}}
+								</optgroup>
+								<optgroup label="Confirmation">
+									{{#each settings.branches.confirmation}}
+										<option value="confirmation-{{this.name}}">{{this.name}}</option>
+									{{/each}}
+								</optgroup>
 							</select>
 						</th>
 						<th>
 							<select id="status-filter">
 								<option value="no-decision">No decision</option>
 								<option value="accepted">Accepted</option>
+								<option value="not-confirmed">Not confirmed</option>
```

Comment:
What's the difference between accepted and not-confirmed? Where does not-confirmed apply to a user (since a single application branch can potentially have multiple confirmation branches)?

Reply:
Accepted includes all accepted applicants regardless of whether or not they have confirmed whereas not confirmed only includes accepted applicants who have not confirmed